statistics/ExecutionSummaryCliParser.go

```diff
@@ -0,0 +1,106 @@
+package statistics
+
+import (
+	"regexp"
+	"strconv"
+)
+
+//ExecutionSummaryCliParser ...
+type ExecutionSummaryCliParser struct {
+}
+
+//Parse ...
+func (instance ExecutionSummaryCliParser) Parse(data string) ExecutionSummary {
+	return ExecutionSummary{
+		RunningTime:   parseRunningTime(data),
+		Throughput:    parseThroughput(data),
+		TotalRequests: parseTotalRequests(data),
+		TotalErrors:   parseTotalErrors(data),
+		Availability:  parseAvailability(data),
+		Bytes: ByteSummary{
+			TotalSent:     parseTotalBytesSent(data),
+			TotalReceived: parseTotalBytesReceived(data),
+		},
+		MeanResponseTime: parseMeanResponseTime(data),
+		MinResponseTime:  parseMinResponseTime(data),
+		MaxResponseTime:  parseMaxResponseTime(data),
+	}
+}
+
+func checkMatchString(match []string) string {
```

Comment:
Shouldn't all the fields on the `Summary` be numeric of some kind... why do a string match?